build/lib/treeshaking.js

```diff
@@ -230,7 +230,7 @@ var NodeColor;
     NodeColor[NodeColor["Black"] = 2] = "Black";
 })(NodeColor || (NodeColor = {}));
 function getColor(node) {
-    return node.$$$color || NodeColor.White;
+    return node.$$$color || 0 /* NodeColor.White */;
```

Comment:
Is this a change in TS output? I would have expected the `NodeColor` enum definition to disappear too?